lib/ret/load_balancing/janus_load_status.ex

```diff
@@ -44,7 +44,7 @@ defmodule Ret.JanusLoadStatus do
         |> Enum.map(fn {a, b, c, d} -> "#{a}.#{b}.#{c}.#{d}" end)
 
       for host <- hosts do
-        %{body: body} = HTTPoison.get!("https://#{host}:7000/meta", [], hackney: [:insecure])
+        %{body: body} = HTTPoison.get!("http://#{host}:7000/meta", [], hackney: [:insecure])
```

Comment:
We can remove the hackney config if we're using http.
```suggestion
        %{body: body} = HTTPoison.get!("http://#{host}:7000/meta")
```